examples/routing_service/routing_service_with_security/dds_security_01/cert/D2_DP.pem

```diff
@@ -0,0 +1,70 @@
+Certificate:
```

Comment:
Is it OK to have both .cnf files and generated certificates in the example?

Reply:
to use referenced certificates from resources/certAuthority